src/features/Cameras/scripts/lib/canvas/checkInput.ts

```diff
@@ -0,0 +1,146 @@
+import { Keypoint } from "@tensorflow-models/pose-detection/dist/types";
+import { Base, Trigger } from "../types/types";
+import { contains } from "./virtualButtons";
+
+/**
+ * まず機能として、　ベース　トリガー　の２つを受け取る必要がある
+ * ベース：　足の位置　どの足がどの場所にあるかを指定する必要がある
+ * トリガー：　手の位置　どの手がどの場所に入るかを指定する
+ *      type1 片手を特定のエリアに入れる
+ *      type2 両手を特定のエリアに入れる
+ *      type3 任意の手を特定のエリアに入れる
+ *
+ * 1.ベースを満たした状態で、
+ * 2.1f手前ではトリガーを発動していない状態で、
+ * 3.トリガーを発動した場合、
+ *  putCharを起動する
+ *
+ */
+
+function putChar(c: String) {
+  console.log(c);
+}
+
+function makeCheckInput(base: Base, trigger: Trigger, character: String) {
```

Comment:
TS の世界では小文字の方が一般的かな
capital の string クラスは予期せぬバグを生みそうだからよほどの理由がない限りあまり使いたくない
```suggestion
function makeCheckInput(base: Base, trigger: Trigger, character: string) {
```

Reply:
そうなんですね！
基本的にはプリミティブ型を使ったほうが良いってことですかね

Reply:
よほどの理由がない限りはプリミティブ使うと思う
大文字の String が有効に働いている場面は（俺は）見たことない

これはバグが怖いから修正してほしい

Reply:
ありがとうございます！
変更しました！